funky-flamingos/rinky_dink/users/views.py

```diff
@@ -0,0 +1,199 @@
+import hashlib
+import os
+
+from django.shortcuts import render, redirect
+from django.contrib import messages, auth
+from django.contrib.auth.decorators import login_required
+from django.core.files.storage import FileSystemStorage
+
+from users.forms import RegistrationForm, TeamRegistrationForm
+from users.models import File, FileGroup, Member, Team
+
+
+def fhandler(request, name):
+    path = "/".join(os.path.dirname(os.path.realpath(__file__)).split("/")[:-1])
+    new_path = path + "/media/" + name
+    with open(new_path, "rb") as f:
+        text = f.read()
+        print(text)
+
+    return render(
+        request, "users/file.html", {"text": text.decode("utf-8"), "name": name}
+    )
+
+
+def index(request):
+    return render(request, "users/index.html")
+
+
+def register(request):
+    if request.method == "POST":
+        form = RegistrationForm(request.POST)
+        if form.is_valid():
+            form.save()
+            print(form.save())
+            messages.success(request, "You have been successfully registered!")
+            return redirect("login")
+    else:
+        form = RegistrationForm()
+
+    return render(request, "users/register.html", {"form": form})
+
+
+@login_required(login_url="/login/")
+def teamregister(request):
```

Comment:
As a rule of thumb, separate words in variable names with underscores, e.g. `team_register`.